test_0.1

esselesse opened the pull request:
plugin.xml пришлось чуть подкорректировать из-за каких-то внутренних багов Eclipse IDE, поэтому здесь в наличии есть и его резервная копия, решил ее коммитить тоже

![image](https://user-images.githubusercontent.com/5176168/77652868-faf34c80-6f7f-11ea-9177-79ebd899b3f3.png)

esselesse committed:
my new stuff

esselesse committed:
еще немного

esselesse committed:
minor debug for pull req possibility

serge-rider commented:
PR/commits shouldn't contain only changes which are not needed for the task. Changes in plugin.xml and other files are not needed.

Also, please use existing messages class, do not create a new one. 
Naming convention is also important.

esselesse committed:
revision due to comments

- tried to revise names of vars according to naming convention
- refactoring: deleted unnecessary 'messages' files, edited
'TaskUIMessages' files

esselesse committed:
deleted unecessary file

esselesse committed:
Delete DBeaver.product.launch

non git-ignored file

esselesse commented:
> Changes in plugin.xml

unfortunately, without them the whole project doesn't compile. I've tried a lot of things, but nothing helps. Stackoverflow and github points that it is a bug somwhere in Eclipse IDE or Maven.

----

I tried to correct according to Your review.
